packages/hardhat-plugin/test/fixture-projects/minimal-new-api/contracts/C.sol

```diff
@@ -0,0 +1,116 @@
+// SPDX-License-Identifier: MIT
```

Comment:
This file can now be deleted. If this fixture-project was added for my decoding code, it can be deleted in its entirety.

We can also move many tests to core/ now